src/securityPolicy.c

```diff
@@ -134,6 +135,7 @@ security_policy_t policyForGetExtendedPublicKeyBulkExport(const bip44_path_t* pa
 		break;
 
 	default:
+		// TODO should multisig keys be allowed here?
```

Comment:
Again, I think we should do the same as with ordinary keys.

I would use `PATH_MULTISIG_*` instead of `PATH_WALLET_*` in the path classification enum for multisig keys.